soco/core.py

```diff
@@ -1261,15 +1223,15 @@ def get_speaker_info(self, refresh=False, timeout=None):
         """Get information about the Sonos speaker.
 
         Arguments:
-        refresh -- Refresh the speaker info cache.
-        timeout -- How long to wait for the server to send
-                   data before giving up, as a float, or a
-                   (`connect timeout, read timeout`_) tuple
-                   e.g. (3, 5). Default is no timeout.
+            refresh(bool): Refresh the speaker info cache.
+            timeout: How long to wait for the server to send
+                data before giving up, as a float, or a
+                `(connect timeout, read timeout)` tuple
+                e.g. (3, 5). Default is no timeout.
 
         Returns:
-        Information about the Sonos speaker, such as the UID, MAC Address, and
-        Zone Name.
+            dict: Information about the Sonos speaker, such as the UID,
+            MAC Address, and Zone Name.
```

Comment:
does that format correctly. I thought the second line "MAC Add...." should be indented.